libres/tests/tmpdir.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef LIBRES_TESTS_TMPDIR_HPP
+#define LIBRES_TESTS_TMPDIR_HPP
+#include <catch2/catch.hpp>
+#include <iostream>
+#include <string>
+#include <filesystem>
+namespace fs = std::filesystem;
```

Comment:
Don't add namespace aliases to header files as they bleed into source files.

Reply:
Ok, thanks.

Should we update this from ecl?

https://github.com/equinor/ecl/blob/master/lib/tests/tmpdir.hpp